src/gui/qgsrelationwidgetregistry.h

```diff
@@ -79,9 +79,25 @@ class GUI_EXPORT QgsRelationWidgetRegistry
      */
     QgsAbstractRelationEditorConfigWidget *createConfigWidget( const QString &widgetType, const QgsRelation &relation, QWidget *parent = nullptr ) const SIP_TRANSFERBACK;
 
+
+    /**
+     * Sets the default editor widget type. Does nothing if the provided widget type is not present.
+     * \param defaultWidgetType The widget type to be used by default.
+     * \since QGIS 3.22
+     */
+    void setDefaultWidgetType( const QString &widgetType );
+
+    /**
+     * Returns the default editor widget type.
+     * \since QGIS 3.22
```

Comment:
this is a bugfix, that should go in 3.20.

Reply:
done